modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/rate/swap/DiscountingKnownAmountPaymentPeriodPricer.java

```diff
@@ -75,6 +75,11 @@ public double accruedInterest(KnownAmountPaymentPeriod period, RatesProvider pro
     return fv * (partialDays / totalDays);
   }
 
+  @Override
+  public double pvbp(KnownAmountPaymentPeriod period, RatesProvider provider) {
+    return 0;
```

Comment:
If the result cannot be calculated, better to throw UnsupportedOperationException